layer.yaml

```diff
@@ -1 +1,23 @@
-includes: ['layer:hadoop-base', 'interface:hdfs', 'interface:mysql']
+includes: ['layer:hadoop-client', 'interface:mysql']
+options:
+  hadoop-client:
+    packages:
+        - 'libmysql-java'
+        - 'mysql-client'
+    users:
+      hive:
+        groups: ['hadoop']
+    dirs:
+      hadoop:
```

Comment:
Can we get rid of the 'hadoop' stanza here?  For one, I don't see any mention of .path('hadoop') in hive.py, and for two, this dir should be created by layer-hadoop-client (indirectly since client includes plugin which defines this dir in its layer opts).